### PR TITLE
[Python Simple Linked List]: Added Example Diagrams

### PR DESCRIPTION
**Pull request by BethanyG**
See [PR #3026](https://github.com/exercism/python/pull/3026) in `exercism/python` for the file context.

- [x] SVG Diagram for Linked List Stack
- [x] SVG Diagram for Array-Based Stack

### CODE REVIEW

**Comment by BethanyG** [comment]
Closing since these are not big files.  Uploading them to the Python repo instead.

**Comment by BethanyG** [comment]
Reopening per comments on https://github.com/exercism/python/pull/3026.

File: .gitattributes [hunk 1]
```diff
@@ -1 +0,0 @@
-*.png filter=lfs diff=lfs merge=lfs -text
```
Comment (SaschaMann) [inline .gitattributes:1]: I think this needs to stay, and you also need to add svg files to be handled by git lfs for this to work as intended.

**Review by SaschaMann — CHANGES_REQUESTED** [review]

**Comment by SaschaMann** [comment]
The files don't say "Stored with LFS", so I'm not sure this has worked correctly. Could you verify that you added them via git lfs?

This info is missing:

grafik

I can take a closer look in the evening otherwise.

**Comment by BethanyG** [comment]
ok.  _hopefully_ I did it correctly this time!

**Pull request merged**